File: World Cuisine/Models/Recipe.cs
namespace World_Cuisine.Models
{
    public class Recipe
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public string ImageUrl { get; set; }
        public int UserId { get; set; }
        public string Ingredient { get; set; }
        public string Instruction { get; set; }
        public User User { get; set; }
    }
}

File: World Cuisine/client/src/components/Recipe.js
import { Card, CardBody, Button } from "reactstrap"
import { useNavigate } from "react-router-dom"

export const Recipe = ({ recipe }) => {

    const navigate = useNavigate()

    const editClick = (e) => {
        e.preventDefault()
        navigate(`/recipe/edit/${recipe.id}`)
    }

    return (
        <Card>
            <p className="text-left px-2">Created by: {recipe.user.fullName}</p>
            <CardBody>
                <strong>{recipe.name}</strong>
                <p>{recipe.description}</p>
                <br />
                <strong>Ingredients:</strong>
                <p>{recipe.ingredient}</p>
                <br />
                <strong>Instructions:</strong>
                <p>{recipe.instruction}</p>
            </CardBody>
            <Button onClick={editClick}>Edit Recipe</Button>
        </Card>
    )
}